src/Azure/Orleans.Streaming.EventHubs/Providers/Streams/EventHub/EventHubAdapterFactory.cs

```diff
@@ -96,7 +96,7 @@ public class EventHubAdapterFactory : IQueueAdapterFactory, IQueueAdapter, IQueu
         /// <summary>
         /// Factory to create a IEventHubReceiver
         /// </summary>
-        protected Func<EventHubPartitionSettings, string, ILogger, ITelemetryProducer, Task<IEventHubReceiver>> EventHubReceiverFactory;
```

Comment:
yeah I can see using API in the new evenhub pkg, a async Task isn't needed anymore. But this is a breaking change indeed. I don't have strong opinion on this, since most people won't be affected. Just want to point that out.